src/WorldStorage/SchematicFileSerializer.cpp

```diff
@@ -7,7 +7,6 @@
 
```

Comment:
The result buffer returned by the compression routines takes up the return value, so they report failure via exceptions. This also means GZipFile reports failure similarly. Since SchematicFileSerializer makes use of both I wanted to avoid mixing two styles of failure handling code.